conditionals.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Conditionals</title>
+  <meta charset="utf-8">
+  <script>
+    var year = prompt("¿how old are you?");
+    if (year>18){
```

Comment:
what if age == 18?

Reply:
if age is == 18 I added a = to the if statement to check if age is above or equal 18.